bindings/protocols/mqtt/index.html

```diff
@@ -321,7 +321,7 @@ <h4>TopicName</h4>
       <p>
         <!-- Generate description form the ontology-->
         
-        The label attached to an Application Message which is matched against the Subscriptions known to the Server. The Server sends a copy of the Application Message to each Client that has a matching Subscription. TopicName is used ony for publication to indicate a single topic, and thus MUST not contain any wildcards.
+        The label attached to an Application Message which is matched against the Subscriptions known to the Server. The Server sends a copy of the Application Message to each Client that has a matching Subscription. TopicName is used only for publication to indicate a single topic, and thus MUST not contain any wildcards.
```

Comment:
Where is the ony in index.template.html?

Reply:
I am guessing that there was a manual fix that should not have happened? Did you change it manually or was it simply a result of the spec generation tool

Reply:
> I am guessing that there was a manual fix that should not have happened?

I see

> Did you change it manually or was it simply a result of the spec generation tool

I manually changed it (without the generation tool)